tox.ini

```diff
@@ -9,10 +9,10 @@ setenv =
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/dev-requirements.txt
-    django111: Django>=1.11,<1.12
 
 # These are the equivalent of 'make ci'.
 commands =
+    django111: pip install 'Django>=1.11,<1.12'
```

Comment:
Note to self: research for a more elegant way to do the same at https://stackoverflow.com/questions/54649374/how-to-require-a-specific-package-version-in-tox

Reply:
The way we've addressed this in repos already following OEP-18 is to post-process the testing dependencies file to remove any existing entry for Django.  Then there's no conflict with the constraints in tox.ini, but we still have it in other requirements files used for setup.py, etc.

Reply:
I will change this to mimic what was done elsewhere to post-process the testing dependencies. Unless I misinterpreted your suggestions, this is the last change before merge and I should not fix anything else. Right?

Reply:
I'm ok with leaving this as is for now; I think the py3 to py36 issue in tox.ini is the last item that I'd particularly like to see changed before final review, squashing commits, and merging.